api/fixtures/example.go

```diff
@@ -471,3 +517,25 @@ func (o ExampleOptions) getServicePublishingStrategyMappingByAPIServerAddress(AP
 		},
 	}
 }
+
+func (o ExampleOptions) EtcdEncryptionKeySecret() *corev1.Secret {
```

Comment:
If we lose this secret because for example someone accidentally deletes it, the etcd data becomes inaccessible, correct?

Reply:
Correct